src/config/partners/anotherblock.ts

```diff
@@ -16,14 +16,52 @@ const anotherblock: Partner = {
   twitter: '@anotherblock_io',
   drops: [
     {
+      startDate: Date.UTC(2023, 7, 12, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
+      endDate: Date.UTC(2023, 7, 14, CAMPAIGN_HOUR, CAMPAIGN_MINUTE, 0, 0),
```

Comment:
Hey, end time is going to be 11:59 PM, not AM, so I believe `CAMPAIGN_HOUR` needs to be updated